website/src/components/Showcase/Card.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import clsx from 'clsx';
+import Link from '@docusaurus/Link';
+import styles from './Card.module.css';
+
+function Card({ className, title, titleFontSize, image, linkGoogle, linkApple, linkGitHub }) {
```

Comment:
@noomorph - Hi Yarik. Why do you pass className in here? I haven't seen that before so I'm just trying to learn. Thanks!

Reply:
Since you're not passing any `className` property to the Card this parameter is currently unused, however, Yarik intention was to allow passing a class-name argument to this component (`<Card className=...`), without this parameter here it won't pass.

See this for example: https://codesandbox.io/s/classname-param-7ctb27

Reply:
We can keep it although it's redundant IMO.. I guess it's a good practice

Reply:
Thanks for the explanation here Asaf. Appreciate it and I understand better now.